liminal/core/environment.py

```diff
@@ -23,7 +23,7 @@
 DEFAULT_DAGS_ZIP_NAME = 'liminal.zip'
 DEFAULT_LIMINAL_HOME = os.path.expanduser('~/liminal_home')
 DEFAULT_PIPELINES_SUBDIR = "pipelines"
-LIMINAL_HOME_PARAM_NAME = "LIMINAL_HOME"
+LIMINAL_HOME_PARAM_NAME = "AIRFLOW__CORE__LIMINAL_HOME"
```

Comment:
This isn't a good name for local env, we should have a different variable to hold this value